src/dependency-manager/src/service-config/base.ts

```diff
@@ -5,7 +5,7 @@ interface RestSubscriptionData {
 
 export interface ServiceParameter {
   description: string;
-  default?: string | number;
+  default?: any;
```

Comment:
No shortcuts when it comes to the specs we support! Make sure to create a proper interface and add the valueFrom syntax as a valid option.